install/activemq/install.yml

```diff
@@ -0,0 +1,17 @@
+id: activemq-integration
+name: ActiveMQ
+title: ActiveMQ monitoring
+description: |
+  Monitor the performance metrics of your Apache ActiveMQ instances in real-time with New Relic APM. New Relic detects your ActiveMQ metrics like queues, brokers, producers, consumers, garbage collection rate, heap memory usage and more.
+
+target:
+  type: integration
+  destination: host
+  os:
+    - linux
+
+install:
+  mode: link
+  destination:
+    url: https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/jmx-monitoring-activemq
```

Comment:
This link 404s

Reply:
Hi @aswanson-nr , 
We have coordinated with @akristen from Documentation team. This docs URL is live now.
Request you to kindly do the needful.
Thank you. 😀